src/modules/bookings/bookings.controller.ts

```diff
@@ -23,4 +24,23 @@ export class BookingsController {
   create(@Body() createBookingDto: CreateBookingDto, @GetAccount('accountId') userId: string) {
     return this.bookingsService.createBooking(createBookingDto, userId);
   }
+
+  @Get()
+  @ApiBearerAuth()
+  @UseGuards(AccessTokenGuard)
+  @ApiOperation({ summary: 'Get user bookings' })
+  @ApiResponse({
+    status: 200,
+    description: 'User bookings successfully received.',
+  })
+  @ApiResponse({ status: 401, description: 'Unauthorized request.' })
+  async getUserBookings(@GetAccount('accountId') userId: string) {
```

Comment:
I think ```accountId``` is more logical. Also, we are using ```accountId``` in other places

Reply:
Fixed